app/component/map/tile-layer/Covid19OpeningHours.js

```diff
@@ -79,6 +79,8 @@ class Covid19OpeningHours {
         return 'poi-closed';
       case 'partial':
         return 'poi-partial';
+      case 'open_adapted':
```

Comment:
You can use several `case` lines like this: https://stackoverflow.com/questions/13207927/switch-statement-multiple-cases-in-javascript

Reply:
ok, cool!